src/uwsgi.ini

```diff
@@ -1,6 +1,6 @@
 [uwsgi]
-uwsgi-socket = 0.0.0.0:8000
-http-socket = 0.0.0.0:8001
+# uwsgi-socket = 0.0.0.0:8000
```

Comment:
Is this the new default? No uwsgi-sockets anymore?

Reply:
This is a controverial decision. The Nginx configuration, at the moment, sets many `proxy_*` parameters since `http-socket` was used in the past. Then it was migrated to `uwsgi-socket` since it's better if we have a uWsgi upstream. However the `proxy_*` parameteres weren't converted, so in this moment they don't have any effect.

We should convert them to the corresponding `[uwsgi_*](http://nginx.org/en/docs/http/ngx_http_uwsgi_module.html)` version, but it requires testing.

I think we should do it, but I would suggest in another dedicated PR. What do you think?

Reply:
Definitely a separate PR